contracts/truffle/Migrations.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity >=0.4.21 <0.6.0;
```

Comment:
Shouldn't ```>=0.4.21``` be ```>=0.5.0``` ?
Why lower version is 0.4.21?

Reply:
This is an auto-generated contract with `truffle init` command. Now it's possible to specify the range in pragma statement. This statement means this contract can compile with all the versions between 0.4.21 and 0.6.0